app-kotlin/src/main/res/values/strings.xml

```diff
@@ -1,3 +1,6 @@
 <resources>
   <string name="app_name">Training Kotlin</string>
+  <string name="attack">Attack</string>
+  <string name="image_of_the_pokemon">image of the pokemon</string>
+  <string name="pokemon_name">Pokemon Name</string>
```

Comment:
Android apps tend to use title case for all their strings. 
```suggestion
  <string name="image_of_the_pokemon">Image of the pokemon</string>
  <string name="pokemon_name">Pokemon name</string>
```

Reply:
Thanks for the suggestion i had resolved this.